vendor/github.com/ethereum/go-ethereum/whisper/whisperv5/filter.go

```diff
@@ -22,8 +22,10 @@ import (
 	"sync"
 
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/message"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/syndtr/goleveldb/leveldb/errors"
```

Comment:
Looks like `goimports` failed here - probably should be stdlib's `errors` package.

Reply:
Oh ya, nice catch!